firebase-perf/src/main/java/com/google/firebase/perf/config/RemoteConfigManager.java

```diff
@@ -69,14 +74,25 @@ private RemoteConfigManager() {
         );
   }
 
-  @VisibleForTesting
   RemoteConfigManager(Executor executor, FirebaseRemoteConfig firebaseRemoteConfig) {
+    this(
+        executor,
+        firebaseRemoteConfig,
+        MIN_APP_START_CONFIG_FETCH_DELAY_MS
+            + new Random().nextInt(RANDOM_APP_START_CONFIG_FETCH_DELAY_MS));
+  }
+
+  @VisibleForTesting
+  RemoteConfigManager(
+      Executor executor, FirebaseRemoteConfig firebaseRemoteConfig, long fetchDelay) {
     this.executor = executor;
     this.firebaseRemoteConfig = firebaseRemoteConfig;
     this.allRcConfigMap =
         firebaseRemoteConfig == null
             ? new ConcurrentHashMap<>()
             : new ConcurrentHashMap<>(firebaseRemoteConfig.getAll());
+    this.appStartTime = getCurrentSystemTimeMillis();
```

Comment:
Is this the app start time? Should this be the time at which the application's onCreate method got called?

Reply:
I did this originally because I was using your iOS PR as a reference, and I assumed that `+ (void)load` was equivalent to constructor, but now I realize it's not, my bad. 

Correct me if I'm wrong but I don't think we currently have a variable or constant anywhere for the application start time. 
Would it be good enough if I made `appStartTime` a static variable in `RemoteConfigManager` and initialize it as it is declared? That way the time should be when the class is initialized, which should be when `public static void getInstance()` is called for the first time (if you trace it backwards it should be called in `FirebasePerfProvider.attachInfo`.

Reply:
This is a tricky one. There is a separate place where the app start time is measured. As much as a possible, we should see if we can rely on that for the start time of the application. This is the valid app start time you should be considering to use.

Please use this measurement instead of creating a new one in RemoveConfigManager.

Reply:
found it and done

Reply:
Sounds good, thanks!